photon-api/src/test/scala/com/linkedin/photon/ml/function/svm/SingleNodeSmoothedHingeLossFunctionTest.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linkedin.photon.ml.function.svm
+
+import breeze.linalg.DenseVector
+import org.testng.Assert.assertEquals
+import org.testng.annotations.Test
+
+import com.linkedin.photon.ml.data.LabeledPoint
+import com.linkedin.photon.ml.normalization.NoNormalization
+import com.linkedin.photon.ml.optimization._
+import com.linkedin.photon.ml.optimization.game.RandomEffectOptimizationConfiguration
+import com.linkedin.photon.ml.util.PhotonNonBroadcast
+
+/**
+ * Unit tests for [[SingleNodeSmoothedHingeLossFunction]].
+ */
+class SingleNodeSmoothedHingeLossFunctionTest {
```

Comment:
Please apply all comments for the `DistributedGLMLossFunctionTest` test to this class as well (except for the `SparkContext`/integration test comments).

Reply:
Done